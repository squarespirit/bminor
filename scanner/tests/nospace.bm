123<456/*hi*/"abc"+"\n"
if(x==y&&__cRaZ__Y)c++;d=5;