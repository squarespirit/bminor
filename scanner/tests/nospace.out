intlit	123
lt	<
intlit	456
comment	/*hi*/
stringlit	"abc"
add	+
stringlit	"\n"
if	if
lparen	(
ident	x
equals	==
ident	y
and	&&
ident	__cRaZ__Y
rparen	)
ident	c
increment	++
semi	;
ident	d
assign	=
intlit	5
semi	;
